### PR TITLE
Issue 186

### LINKED ISSUE #186: Use ga4gh.vrsatile.pydantic models
https://pypi.org/project/ga4gh.vrsatile.pydantic/

### PR DESCRIPTION
**Pull request by korikuzma**
Closes #186

### CODE REVIEW

File: therapy/etl/hemonc.py
Comment (jsstevenson) [inline therapy/etl/hemonc.py:179]: Let's just push the disease norm update through to PyPI and then we can update this/pin the new version in the Pipfile

**Review by jsstevenson — CHANGES_REQUESTED** [review]

File: therapy/etl/hemonc.py
Comment (korikuzma) [reply therapy/etl/hemonc.py:179]: Ya I was going to wait until disease got merged!

File: Pipfile
Comment (jsstevenson) [inline Pipfile:39]: should pin the disease requirement at >= 0.2.9 since there's a breaking change

**Review by jsstevenson — DISMISSED** [review]

**Review by jsstevenson — APPROVED** [review]

**Pull request merged**
